### ISSUE TITLE
gamecenter json not updating

### ISSUE THREAD

**Issue by mesee298** [BurntSushi/nflgame#185]
I had a problem that the Dallas and GB game didn't have correct information.  If I ran it on my computer it would run fine, but if I ran on another it wouldn't show all the information.  I realized that the 2015121313.json.gz had a datestamp of 12/13/2015 3:05 PM (CST).  Once I renamed it to .sav and ran the scoring again, it created a new file and it was correct.

Is there something that I need to do to force nflgame to get a new json each time to ensure it is the latest and greatest?

**Comment by BurntSushi** [comment]
If the game is over, no new JSON will ever be retrieved. Simply delete the JSON and it will refetch automatically.

**Comment by ochawkeye** [comment]
No, I had the same thing happen.  This was a glitch from the json feed yesterday for that game where it reported that the game had gone final at halftime.  

`nflgame` caches only games that have completed, so if the mis-reported final score had not happened, `nflgame` would have pulled the json data from nfl.com every time it was requested until it did go final.

For what it's worth, I can count on one hand the number of times I have seen something like this happen in the almost 3 years I have been using `nflgame`.

**Comment by mesee298** [comment]
Okie Dokie.. Thanks to both of your for the quick response.

**Comment by patmun** [comment]
I experienced something similar with a game between Seatle and Miami this week (2016 wk 1). The game ended on a play (a fumble on the QB) by C.Avril. All sites (NFL.com, ESPN, etc) report this play as a sack credited to this player. However, the json from NFL.com makes no mention of this play (or at least of this player for the last play of the game). 

I deleted the corresponding json.gz file so it gets regenerated from the website, but same result. Here is a link to the NFL.com json feed. 
http://www.nfl.com/liveupdate/game-center/2016091109/2016091109_gtd.json
I'm I just unlucky or does this happens often? And is there a way to ask the to correct the feed?
